streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -615,7 +615,7 @@ public void shouldUseSpecifiedNameForTransformValuesWithKey() {
     }
 
     @Test
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "deprecation"})
```

Comment:
Seems we should add a corresponding test for `split()` ? (Same for other test classes.)

Reply:
done